src/Operators/divergence_operators.jl

```diff
@@ -36,39 +22,13 @@ end
 
 Calculates the 2D divergence ∂x u + ∂y v via
 
-    1/V * [δxᶜᵃᵃ(Ax * u) + δyᵃᶜᵃ(Ay * v)]
+    1/Azᶜᶜᵃ * [δxᶜᵃᵃ(Δy * u) + δyᵃᶜᵃ(Δx * v)]
```

Comment:
In comparing this with 2.138 in the MITgcm docus, I see that one difference is they have h's and here we don't.  Is it easy to understand why there is a different notation?

Reply:
Yes! The `h` is the "fractional height" of the cell. It's 1 in water, 0 underneath the ocean, and between 0 and 1 in a "partial cell" (at most one grid point per column). We don't have bathymetry so we set all `h=1`.

Reply:
Nice!